code/modules/error_handler/error_viewer.dm

```diff
@@ -112,7 +112,6 @@ var/global/datum/ErrorViewer/ErrorCache/error_cache = null
 
 	// Show the error to admins with debug messages turned on, but only if one
 	//  from the same source hasn't been shown too recently
-	// (Also, make sure config is initialized, or log_debug will runtime)
```

Comment:
The comment applies to the line directly below it. The one where config's existence is checked. It explains why config's existence is checked.